plugins/postcss-initial/test/multivalue.expect.css

```diff
@@ -0,0 +1,7 @@
+div {
+	background-repeat: initial initial;
+}
+
+div {
+	background-repeat: initial no-repeat;
+}
```

Comment:
`initial` is a css wide keyword.

Using this in a shorthand together with other values is just invalid and must be ignored.

The old plugin didn't ignore these and did weird other things.

I am following the spec and so ignoring as expected.